wpninjas/ninja-forms
# Settings no longer re-render by default. Setting types can still spec…

…ify that they would like to re-render. Also added a class to advanced boxes. 

To Test: 

- Edit as many settings as you can. Make sure that the setting drawer renders all the settings properly.

Closes #2108.
